promgen/tests/test_models.py

```diff
@@ -34,3 +36,32 @@ def test_validators(self):
             validators.metricname(
                 "[[this.$el.ownerDocument.defaultView.alert(1337)]]",
             )
+
+    @mock.patch("django.dispatch.dispatcher.Signal.send")
+    def test_custom_save_rule(self, mock_post):
```

Comment:
```suggestion
    def test_rule_annotation(self, mock_post):
```

All tests inside this function are related to the `rule` annotation, so let's use a more appropriate name for it.